test/corpus/comments.txt

```diff
@@ -14,7 +14,7 @@ One line comments
 -- MODEL
 -- update
 -- more words
--- note: (isOkay x || any isOkay xs) would not get TCO
+-- (isOkay x || any isOkay xs) would not get TCO
```

Comment:
Did you  mean to edit this too?

Reply:
Nope sorry! I did a regex find and replace to turn the `tree-sitter parse` output into a test case.